### PR TITLE
Authentication to Status server

### PR DESCRIPTION
**Pull request by vogdb**

### CODE REVIEW

File: lib/status-server.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+};
+
+/**
+ * @param {http.IncomingMessage} request
+ * @returns {boolean}
+ */
+StatusServer.prototype._isAuthenticated = function(request) {
+  if (this._secret) {
+    var header = (request.headers && request.headers['authorization']) || '';
+    var token = header.split(/\s+/).pop();
```
Comment (tomaszdurka) [inline lib/status-server.js]: Not liking this so much :)

DIviding by whitespace is hacky.

Comment (tomaszdurka) [inline lib/status-server.js]: ~~What about having special header with token as full-value? Not sure what are the patterns here.~~

image

I see that it's the way. Let's make a pregmatch for that then.

**Comment by vogdb** [comment]
@tomaszdurka please review

**Comment by tomaszdurka** [comment]
lgtm

**Pull request merged**